components/input/Input.tsx

```diff
@@ -47,20 +47,18 @@ export interface InputProps {
   addonBefore?: React.ReactNode;
   addonAfter?: React.ReactNode;
   onPressEnter?: React.FormEventHandler<any>;
-  onKeyDown?: React.FormEventHandler<any>;
-  onChange?: React.FormEventHandler<any>;
   onClick?: React.FormEventHandler<any>;
   onFocus?: React.FormEventHandler<any>;
   onBlur?: React.FormEventHandler<any>;
   autosize?: boolean | AutoSizeType;
-  autoComplete?: 'on' | 'off';
   style?: React.CSSProperties;
   prefix?: React.ReactNode;
   suffix?: React.ReactNode;
-  spellCheck?: boolean;
 }
 
-export default class Input extends Component<InputProps, any> {
+export type HTMLInputProps = React.HTMLProps<HTMLInputElement>;
+
+export default class Input extends Component<InputProps & HTMLInputProps, any> {
```

Comment:
Actually, we had mix input and textarea into Input, see: https://github.com/ant-design/ant-design/blob/5d6439f15b9213942a4b8892fe6aadecabb76490/components/input/Input.tsx#L252-L259